src/pybind/mgr/volumes/fs/async_cloner.py

```diff
@@ -221,11 +221,13 @@ def set_quota_on_clone(fs_handle, clone_volumes_pair):
 
 def do_clone(fs_client, volspec, volname, groupname, subvolname, should_cancel):
     with open_volume_lockless(fs_client, volname) as fs_handle:
-        with open_clone_subvolume_pair(fs_client, fs_handle, volspec, volname, groupname, subvolname) as clone_volumes:
-            src_path = clone_volumes[1].snapshot_data_path(clone_volumes[2])
-            dst_path = clone_volumes[0].path
+        with open_clone_subvolume_pair(fs_client, fs_handle, volspec, volname,
+                                       groupname, subvolname)
+            as (subvol0, subvol1, subvol2):
```

Comment:
The `open_clone_subvolume_pair` returns (clone_subvol, source_subvol, src_snapname). Please name it accordingly to improve readability.

Reply:
Sure.